MetaMask/metamask-extension
# fix: Remove Metamask Fees line from fee details component.

## **Description**
We should remove the `MetaMask fee` line from fee details component. Main reason for this is the concern with a potential backslash due to our users missunderstanding the purpose of this inclusion.

We'll revisit this during the user research around gas that we'll run during Q1.

## **Related issues**

Fixes: [#1818](https://github.com/MetaMask/MetaMask-planning/issues/1818)

## **Manual testing steps**

1. Launch MM
2. Open testdapp and perform an EIP1559 transaction
3. Expand the fee details section on MM window
4. You should see a `Metamask fees` line in fee details
5. Checkout this branch and build
6. Repeat 2-4
7. You shouldn't see a fee details section

## **Screenshots/Recordings**

### **Before**

<img width="352" alt="Screenshot 2023-12-11 at 19 08 45" src="https://github.com/MetaMask/metamask-extension/assets/44811/4fd44954-2bdb-4c60-969b-28153486fd04">


### **After**
<img width="342" alt="Screenshot 2023-12-11 at 19 09 27" src="https://github.com/MetaMask/metamask-extension/assets/44811/d1fa9c72-47cc-4f13-8300-4808f55eeb6f">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
